test-clients/src/main/java/com/hedera/services/legacy/core/CommonUtils.java

```diff
@@ -339,45 +333,6 @@ public static Object convertFromBytes(byte[] bytes) {
     return rv;
   }
 
-  /**
-   * Returns a short form of a string in the form of display first and last n characters.
-   */
-  public static String shortForm(String string, int n) {
-    int d = 2 * n;
-    int len = string.length();
-    if (string.length() <= d) {
-      return string;
-    }
-    String rv = "length=" + len + ": content=" + string.substring(0, n) + " ... " + string
-        .substring(len - d, len);
-    return rv;
-  }
-
-  /**
-   * Serialize a Java object into Json string.
-   *
-   * @param obj Java object
-   * @return Json string
-   */
-  public static String serialize2Json(Object obj) {
-    Gson gsonObj = new Gson();
-    String json = gsonObj.toJson(obj);
-    return json;
-  }
-
-  /**
-   * Deserialize a Json string into a Java object.
-   *
-   * @param json Java object
-   * @return Json string
-   */
-  @SuppressWarnings({"unchecked"})
-  public static List<String> deserializeListOfStringFromJson(String json) {
-    Gson gsonObj = new Gson();
-    List<String> rv = gsonObj.fromJson(json, ArrayList.class);
-    return rv;
-  }
-
```

Comment:
These are removed because we don't use them anymore ?? These changes are not related to GRPC right

Reply:
True,, those were unused and apparently `io.grpc:*:1.33` dropped the transitive dependency that brought `Gson` into our classpath. So I needed to delete the unused methods to compile.